packages/readmodel/src/notificationConfigReadModelService.ts

```diff
@@ -133,7 +134,21 @@ export function notificationConfigReadModelServiceBuilder(
               userNotificationConfigInReadmodelNotificationConfig.tenantId,
               tenantIds
             ),
-            eq(enabledNotificationTable.notificationType, notificationType)
+            eq(enabledNotificationTable.notificationType, notificationType),
+            match(notificationChannel)
+              .with("inApp", () =>
+                eq(
+                  userNotificationConfigInReadmodelNotificationConfig.inAppNotificationPreference,
+                  true
+                )
```

Comment:
I think you can just write
```suggestion
                userNotificationConfigInReadmodelNotificationConfig.inAppNotificationPreference,
```
since it's a boolean column